src/app.ts

```diff
@@ -30,6 +31,11 @@ import { apolloServer } from './routes/graphql/index.js';
 import { openApiRouter } from './routes/openapi.js';
 import * as system from './system.js';
 
+if (config.APEX_TX_ID !== undefined && config.APEX_ARNS_NAME !== undefined) {
+  log.error('Cannot set both APEX_TX_ID and APEX_ARNS_NAME');
```

Comment:
nit:
I'm thinking if it's immediately obvious or not that it's a user configuration error, could be written: Please only specify env-var APEX_TX_ID or env-var APEX_ARNS_NAME but not both.

Reply:
I agree with the general suggestion, but I would go with "APEX_TX_ID and APEX_ARNS_NAME are mutually exclusive but both are set." as the message.

Reply:
[`056e8c0` (#344)](https://github.com/ar-io/ar-io-node/pull/344/commits/056e8c088e7971abb2d68ef117ca6d21de4939c6)